content/3_Silver/Conclusion.problems.json

```diff
@@ -31,10 +30,11 @@
       "uniqueId": "cf-1174D",
       "name": "Ehab and the Expected XOR Problem",
       "url": "https://codeforces.com/contest/1174/problem/D",
+
       "source": "CF",
-      "difficulty": "Normal",
+      "difficulty": "Easy",
       "isStarred": false,
-      "tags": ["Prefix Sums"],
+      "tags": ["Binary Search"],
```

Comment:
why these changes?

Reply:
Idk I just copied and pasted from CPI/usaco-guide. You can fix it yourself.

Reply:
ok i think i fixed it